src/content/docs/en/guides/images.mdx

```diff
@@ -4,159 +4,673 @@ description: Learn how to use images in Astro.
+Currently, the built-in assets feature does not include a `<Picture />` component.
 
-The `src` attribute is required, and its format will depend on where your images are stored, and whether you have enabled experimental support for assets:
+Instead, you can [generate images or custom components using `getImage()`](/en/guides/images/#generating-images-with-getimage) that use the HTML image attributes `srcset` and `sizes` or the `<picture>` tag [for art direction or to create responsive images](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images#art_direction).
+
+#### Properties
+
+##### src (required)
+
+The format of the `src` value of your image file depends on where your image file is located:
+
+- **Local images in `src/`** - you must **also import the image** using a relative file path or configure and use an [import alias](/en/guides/aliases/). Then use the import name as the `src` value:
+
+  ```astro title="src/pages/index.astro" "myImportedImage" "{myImportedImage}"
+  ---
+  import { Image } from 'astro:assets';
+  import myImportedImage from `../assets/my-local-image.png`
+  ---
+  <Image src={myImportedImage}  alt="descriptive text" />
+  ```
+
+- **Images in the `public/` folder** -  use the image's **file path relative to the public folder**:
+
+  ```astro title="src/pages/index.astro" '"/images/my-public-image.png"'
+  ---
+  import { Image } from 'astro:assets';
+  ---
+  <Image 
+    src="/images/my-public-image.png"
+    alt="descriptive text"
+    width="200"
+    height="150" />
+  ```
+
+- **Remote images** - use the image's **full URL** as the property value:
+
+  ```astro title="src/pages/index.astro" '"https://example.com/remote-image.jpg"'
+  ---
+  import { Image } from 'astro:assets';
+  ---
+  <Image 
+    src="https://example.com/remote-image.jpg"
+    alt="descriptive text"
+    width="200"
+    height="150" />
+  ```
+
+##### alt (required)
+
+Use the required `alt` attribute to provide a string of [descriptive alt text](https://www.w3.org/WAI/tutorials/images/) for images.
+
+If an image is merely decorative (i.e. doesn't contribute to the understanding of the page), set `alt=""` so that screen readers and other assistive technologies know to ignore the image.
+
+##### width and height (required for `public/` and remote images)
+
+These properties define the dimensions to use for the image.
+
+When using local images in their original aspect ratio, the `width` and `height` can be automatically inferred from the source file and are optional.
+
+However, both of these properties are required for remote images and images stored in your `public/` folder as Astro is unable to analyze these files.
+
+##### format 
+
+You can optionally state the [image file type](https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#common_image_file_types) output to be used.
+
+By default, the `<Image />` component will produce a `webp` file.
```

Comment:
When we mention other formats we always add the dot in the start, shouldn't we follow this for image formats as well?
```suggestion
By default, the `<Image />` component will produce a `.webp` file.
```